src/BaseTemplates/EmptyWeb/Startup.cs

```diff
@@ -22,8 +23,13 @@ public void Configure(IApplicationBuilder app)
         {$if$ ($aspnet_useplatformhandler$ == false)
```

Comment:
I was leaning toward leaving the code conditionals in there for the service fabric scenario, so that the code doesn't even get invoked.